doc/source/examples/03_modeling/design_parameters.mystnb

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
It might be interesting to add an extra body with a fixed size so that after modifying the parameters, when you request plotting, you can visually see the difference with respect to the first plot. This is just a suggestion though - feel free to disregard =)

Reply:
Done, I replaced the file with another object on the side as a scale